src/Apps/ViewingRoom/Routes/Statement/__tests__/ViewingRoomStatementRoute.test.tsx

```diff
@@ -1,5 +1,130 @@
-describe.skip("ViewingRoomStatementRoute", () => {
-  it("todo", () => {
-    //
+import React from "react"
+import { MockBoot, renderRelayTree } from "DevTools"
+import { graphql } from "react-relay"
+import { ViewingRoomStatementRoute_Test_QueryRawResponse } from "__generated__/ViewingRoomStatementRoute_Test_Query.graphql"
+import { Breakpoint } from "@artsy/palette"
+import { ViewingRoomStatementRouteFragmentContainer } from "../ViewingRoomStatementRoute"
+import { ViewingRoomStatmentRouteFixture } from "Apps/ViewingRoom/__tests__/Fixtures/ViewingRoomStatementRouteFixture"
```

Comment:
I tend to prefer putting once-used fixtures in the file in which they're used. Makes it a little longer but means we don't need to `export` and risk keeping old fixture files around that aren't used.

Reply:
Thats a good point, yeah right now things are all spread out. Most `Apps/*` fixtures actually ended up `Apps/__tests__/Fixtures/*`, kinda far away, but it makes sense to colocate everything. Will update in follow-up.